code/modules/awaymissions/mission_code/snowdin.dm

```diff
@@ -1,42 +1,223 @@
+	icon_state = "awaycontent26"
+
+/area/awaymission/snowdin/post/cavern1
+	name = "Snowdin Outpost - Cavern Outpost 1"
+	icon_state = "awaycontent27"
+
+/area/awaymission/snowdin/post/cavern2
+	name = "Snowdin Outpost - Cavern Outpost 2"
+	icon_state = "awaycontent28"
+
+/area/awaymission/snowdin/post/mining_dock
+	name = "Snowdin Outpost - Underground Mine Post"
+	icon_state = "awaycontent22"
+
+/area/awaymission/snowdin/post/broken_shuttle
+	name = "Snowdin Outpost - Broken Transist Shuttle"
+	icon_state = "awaycontent20"
+	requires_power = FALSE
 
 /area/awaymission/snowdin/igloo
 	name = "Snowdin Igloos"
-	icon_state = "awaycontent3"
+	icon_state = "awaycontent14"
+	dynamic_lighting = DYNAMIC_LIGHTING_FORCED
 
 /area/awaymission/snowdin/cave
 	name = "Snowdin Caves"
-	icon_state = "awaycontent4"
+	icon_state = "awaycontent15"
+	dynamic_lighting = DYNAMIC_LIGHTING_FORCED
+
+/area/awaymission/snowdin/cave/cavern
+	name = "Snowdin Depths"
+	icon_state = "awaycontent23"
+
+/area/awaymission/snowdin/cave/mountain
+	name = "Snowdin Mountains"
+	icon_state = "awaycontent24"
+
 
 /area/awaymission/snowdin/base
 	name = "Snowdin Main Base"
-	icon_state = "awaycontent5"
+	icon_state = "awaycontent16"
+	dynamic_lighting = DYNAMIC_LIGHTING_ENABLED
 	requires_power = TRUE
 
 /area/awaymission/snowdin/dungeon1
 	name = "Snowdin Depths"
-	icon_state = "awaycontent6"
+	icon_state = "awaycontent17"
+	dynamic_lighting = DYNAMIC_LIGHTING_ENABLED
 
 /area/awaymission/snowdin/sekret
 	name = "Snowdin Operations"
-	icon_state = "awaycontent7"
+	icon_state = "awaycontent18"
+	dynamic_lighting = DYNAMIC_LIGHTING_ENABLED
 	requires_power = TRUE
 
-
-
+/area/shuttle/snowdin/elevator1
+	name = "Excavation Elevator"
+
+/area/shuttle/snowdin/elevator2
+	name = "Mining Elevator"
+
+//shuttle console for elevators//
+
+/obj/machinery/computer/shuttle/snowdin/mining
+	name = "shuttle console"
+	desc = "A shuttle control computer."
+	icon_screen = "shuttle"
+	icon_keyboard = "tech_key"
+	light_color = LIGHT_COLOR_CYAN
+	shuttleId = "snowdin_mining"
+	possible_destinations = "snowdin_mining_top;snowdin_mining_down"
+
+
+//liquid plasma!!!!!!//
+
+/turf/open/lava/plasma
+	name = "liquid plasma"
+	desc = "A flowing stream of chilled liquid plasma. You probably shouldn't get in."
+	icon_state = "liquidplasma"
+	initial_gas_mix = "o2=0;n2=82;plasma=24;TEMP=120"
+	baseturfs = /turf/open/lava/plasma
+	slowdown = 2
+
+	light_range = 3
+	light_power = 0.75
+	light_color = LIGHT_COLOR_PURPLE
+
+/turf/open/lava/plasma/attackby(obj/item/I, mob/user, params)
+	var/obj/item/reagent_containers/glass/C = I
+	if(C.reagents.total_volume >= C.volume)
+		to_chat(user, "<span class='danger'>[C] is full.</span>")
+		return
+	C.reagents.add_reagent("plasma", rand(5, 10))
+	user.visible_message("[user] scoops some plasma from the [src] with \the [C].", "<span class='notice'>You scoop out some plasma from the [src] using \the [C].</span>")
+
+/turf/open/lava/plasma/burn_stuff(AM)
```

Comment:
should make you a plasmaman tbh.
slowly replacing the species of your limbs, one by one, once you get all limbs converted set the human's actual species and boom, plasboi.

leaves a nice story, "what's wrong with your arm?" "Snowdin, snowdin is what's wrong"

Reply:
what the fuck

Reply:
i honestly didn't know what to do with plasma and the cold was just a temp effect but the whole 'slowly turning into a plasmameme' is pretty nifty so I'm totally gonna do that now.